composer.json

```diff
@@ -45,6 +45,7 @@
         "jms/di-extra-bundle": "~1.7",
         "sensio/generator-bundle": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
+        "phpunit/phpunit": "^5.3",
```

Comment:
Because I can! (Honest: Please see conversation on top of this site, behind this line "greg0ire and 1 other commented on an outdated diff 2 days ago") Dont know how to link direct to this.